### ISSUE TITLE
[hold for payment 2023-08-30] IOS -  Onfido screen present multiple time when app pushed to Background

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#23642]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to setting
2. Go to any workspace
3. Setup Bank account
4. On last step click Save & continue
5. It will present Onfido | Real identity screen
6. Push app in background
7. Again open the app

## Expected Result:
Onfido | Real identity screen should be presented only once

## Actual Result:
Onfido | Real identity screen presented multiple time when app goes in background and again opened

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [x] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.42-26
**Reproducible in staging?:** n/a
**Reproducible in production?:** n/a
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/46abcea6-b0b0-4db9-95c1-bedc49b3bf59

**Expensify/Expensify Issue URL:**
**Issue reported by:** @DinalJivani
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1689948418893939

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~013b5a7fa2bf8b3fcc</li>
        <li>Upwork Job ID: 1684610566252363776</li>
        <li>Last Price Increase: 2023-07-27</li>
<li>Automatic offers: </li>
<ul>
<li>ginsuma | Contributor | 25908892</li>
<li>DinalJivani | Reporter | 25908893</li>
</ul></li>
    </ul>
</details>

**Comment by joekaufmanexpensify** [comment]
I can reproduce this but I think we're using an Onfido SDK for this, so not sure this is something we can fix. Checking.

**Comment by joekaufmanexpensify** [comment]
Bumped in Slack

**Comment by joekaufmanexpensify** [comment]
Awaiting proposals

**Comment by jaskarwal** [comment]
Hi @joekaufmanexpensify ,

<img width="357" alt="Screenshot 2023-07-30 at 12 19 02 AM" src="https://github.com/Expensify/App/assets/105599495/cae9cba7-1e17-4cdc-a057-818edcd86db1">
I am trying  to resolve the issue but I am not able to attach to debugger on iOS simulator. Is there any other branch to check ?

**Comment by joekaufmanexpensify** [comment]
@allroundexperts any thoughts on the above question?

**Comment by joekaufmanexpensify** [comment]
Proposal pending review

**Comment by joekaufmanexpensify** [comment]
Proposal pending final review from Nikki

**Comment by NikkiWines** [comment]
Oops, sorry for the delay here. Agreed that @ginsuma's proposal looks good 👍

**Comment by joekaufmanexpensify** [comment]
Payment is due on 2023-08-30

**Comment by joekaufmanexpensify** [comment]
This one took longer than 9 business days, so there would be a 50% penalty here. This means we need to make the following payments: 

- @allroundexperts - $500 for C+ review (reduced from $1,000). Paid via NewDot. 
- @ginsuma - $500 for PR (reduced from $1,000). Paid via Upwork. 
- @DinalJivani - $250 for reporting the bug. Paid via Upwork.

**Comment by joekaufmanexpensify** [comment]
Got it. I will review further!

**Comment by joekaufmanexpensify** [comment]
Ah, thanks for flagging that. Yeah, I see @NikkiWines approved on 2023-08-14, but did not merge until 2023-08-17. If we remove those three days then the time here was 8 business days, so no penalty would be applied.

**Comment by joekaufmanexpensify** [comment]
Just edited the payment message to reflect that there will no penalty here.

**Comment by joekaufmanexpensify** [comment]
@ginsuma $1,000 sent and contract ended!

**Comment by joekaufmanexpensify** [comment]
@DinalJivani $250 sent and contract ended!

**Comment by joekaufmanexpensify** [comment]
@allroundexperts could you please request $1,000 via NewDot and confirm here once complete?

**Comment by joekaufmanexpensify** [comment]
Great, thanks for confirming!

**Comment by joekaufmanexpensify** [comment]
Going to close this one out for now. If your request is not paid within 7 days, please let me know and I'll look into it.

**Comment by joekaufmanexpensify** [comment]
As an FYI for the individual paying, payment summary message is [here. ](https://github.com/Expensify/App/issues/23642#issuecomment-1698122881)

**Comment by JmillsExpensify** [comment]
Reviewed the details for @allroundexperts. $1,000 payment approved based on BZ summary.

**Comment by kavimuru** [comment]
@joekaufmanexpensify @allroundexperts Tester was able to reproduce it today. Should we need a new ticket?

https://github.com/Expensify/App/assets/43996225/bab09c81-ca7a-4bec-8639-d32550766d0c

**Comment by joekaufmanexpensify** [comment]
@kavimuru If this is broken again now, wouldn't it be a regression? If so, I'd think we'd fix it on the issue that introduced the regression.
